lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2212,6 +2212,73 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :type => Integer,
           :allowed_from_server => false,
           :description => 'This value represents the total amount of memory available to the host (not the process), in mebibytes (1024 squared or 1,048,576 bytes).'
+        },
+        # security agent
+        :'security.agent.enabled' => {
+          :default => false,
+          :external => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => 'If `true`, the security agent is loaded'
+        },
+        :'security.enabled' => {
+          :default => false,
+          :external => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => 'If `true`, the security agent is activated'
```

Comment:
```suggestion
          :description => 'If `true`, the security agent is started'
```

We have a concept of the language agent "starting" rather than "activating". I think we should use a familiar term here.

Reply:
Committed, but these params will all change once the agent spec is finalized.